lib/runner/VirtualDeviceInvoker.js

```diff
@@ -90,6 +90,17 @@ module.exports = class VirtualDeviceInvoker extends Invoker {
         }
     }
 
+    beforeTest(test) {
+        maxResponseWaitTime = test.testSuite.maxAsyncE2EResponseWaitTime;
```

Comment:
Is max response wait time definitely per interaction? I ran some tests recently and it definitely seemed to be the total time for ALL interactions. I thought this was not supposed to be the case.

Reply:
it's done per interaction. One thing that we noted with Ken though was that since the server executes the speech to text while the next interaction is being sent, the times will differ from the server specially when using the PAUSE event, so even when the server has PAUSE for 10 seconds, for example, the client will show less than that, and have some of that time in the previous utterance.

Reply:
Okay - is it possible I saw different behavior due to using an older version? Was there a change made to this behavior recently?

Reply:
the only change made to that functionality is the one in this PR to use the PAUSE time regardless of the max response wait time. 
Do you have the asyncMode: true property set up? Our default still does asyncMode: false if you don't set the variable explicitly.

Reply:
Hmm...okay, while I will check on this when I next run tests. My observations on this were with the Twilio functionality, which I thought only worked in async mode. But sounds like I need to try it again to see.